avocado/plugins/resolvers.py

```diff
@@ -44,7 +45,8 @@ def resolve(reference):
 
         return ReferenceResolution(reference,
                                    ReferenceResolutionResult.SUCCESS,
-                                   [Runnable('exec-test', reference)])
+                                   [Runnable('exec-test', reference,
+                                             settings.as_dict('^runner'))])
```

Comment:
We should use the same filter used in `avocado/core/suite.py` here too, that is, `r'^runner\.'`.